packages/discord.js/src/structures/PartialGroupDMChannel.js

```diff
@@ -44,6 +46,36 @@ class PartialGroupDMChannel extends BaseChannel {
      * @type {PartialGroupDMMessageManager}
      */
     this.messages = new PartialGroupDMMessageManager(this);
+
+    if ('owner_id' in data) {
+      /**
+       * The user id of the owner of this Group DM Channel
+       * @type {?Snowflake}
+       */
+      this.ownerId = data.owner_id;
+    } else {
+      this.ownerId ??= null;
+    }
+
+    if ('last_message_id' in data) {
+      /**
+       * The channel's last message id, if one was sent
+       * @type {?Snowflake}
+       */
+      this.lastMessageId = data.last_message_id;
+    } else {
+      this.lastMessageId ??= null;
+    }
+
+    if ('last_pin_timestamp' in data) {
+      /**
+       * The timestamp when the last pinned message was pinned, if there was one
+       * @type {?number}
+       */
+      this.lastPinTimestamp = Date.parse(data.last_pin_timestamp);
```

Comment:
This will return `NaN` for null last_pin_timestamp

```suggestion
      this.lastPinTimestamp = data.last_pin_timestamp ? Date.parse(data.last_pin_timestamp) : null;
```